src/features/collections/data_products/Biolog.tsx

```diff
@@ -19,7 +19,7 @@ import { useAppDispatch, useBackoffPolling } from '../../../common/hooks';
 import { formatNumber } from '../../../common/utils/stringUtils';
 import { useAppParam } from '../../params/hooks';
 import classes from '../Collections.module.scss';
-import { useGenerateSelectionId } from '../collectionsSlice';
+import { /* useMatchId, */ useGenerateSelectionId } from '../collectionsSlice';
```

Comment:
Maybe delete vs comment? Not sure why it's commented

Reply:
Ah yes, that was part of the other RE2022-236 work. We'll need `useMatchId` for that, but I'll remove it for the purposes of this PR.